src/styles/index.scss

```diff
@@ -1,3 +1,89 @@
-body {
+$product-font-color: #060b35;
+
+@import 'stars';
+
+* {
   margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
```

Comment:
it's allowed to use only `box-sizing: border-box;` at `*` selector. 
Reset `margin` and `padding` at tag selectors instead.

And it's better to create `common.scss` file and move `*` and other tag selectors to this file